docs/mast/mast.rst

```diff
@@ -868,6 +868,97 @@ To access sector information at a particular location there is  `~astroquery.mas
                 tess-s0010-1-4     10      1   4
 
 
+Zcut
+====
+
+Zcut for MAST allows users to request cutouts from various Hubble deep field surveys. The cutouts are
+returned as full-frame image (FFI) cutouts. This tool can be accessed in Astroquery by using the Zcut
+class.
```

Comment:
The cutouts are returned as full-frame image (FFI) cutouts.-> The cutouts can be returned as either fits or image files (jpg and png are supported).

Add: "The list of supported deep feel surveys can be found here: https://mast.stsci.edu/zcut/"